types/stacktrace-js/index.d.ts

```diff
@@ -47,6 +48,13 @@ declare namespace StackTrace {
   export function get(options?: StackTraceOptions): Promise<StackFrame[]>;
 
   /**
+   * Get a backtrace from invocation point synchronously.
+   * @param options Options Object
+   * @return Array[StackFrame]
+   */
+  export function getSync(options?: { filter?: (stackFrame: StackFrame) => boolean }): StackFrame[]
```

Comment:
I am a bit surprised this is not the `StackTraceOptions` But I also see some inconsistent in the original library 

- https://github.com/stacktracejs/stacktrace.js/blob/1e3acf53c358e01a05ea9f68a445f7f76e1f93f4/stacktrace-js.d.ts#L83 

- https://github.com/stacktracejs/stacktrace.js/blob/1e3acf53c358e01a05ea9f68a445f7f76e1f93f4/README.md#stacktracegetsyncoptional-options--arraystackframe